www/src/helpers/client.js

```diff
@@ -32,16 +32,15 @@ const retryLink = new RetryLink({
   },
 })
 
-const authLink = setContext((_, { headers }) => {
+const authLink = setContext(() => {
   const token = fetchToken()
-  const authHeaders = token ? { authorization: `Bearer ${token}` } : {}
 
   return {
-    headers: Object.assign(headers || {}, authHeaders),
+    headers: token ? { authorization: `Bearer ${token}` } : {},
```

Comment:
aren't there other headers at times?  We could do `{...(headers || {}), ...authHeaders}` I suppose to avoid use of Object.assign